vulture/core.py

```diff
@@ -255,7 +259,8 @@ def by_size(item):
         return sorted(confidently_unused,
                       key=by_size if sort_by_size else by_name)
 
-    def report(self, min_confidence=0, sort_by_size=False):
+    def report(self, min_confidence=0, sort_by_size=False,
+               make_whitelist=False):
```

Comment:
We also have to only report a subset of unused items when `--make-whitelist` is active: e.g., unreachable code can't be added to a whitelist. Therefore, I recommend adding a new method `Vulture.make_whitelist()` instead of adding the functionality to `Vulture.report()`.

Reply:
Hey @jendrikseipp, I saw your email just now after pushing the commit. Making amendments now.